examples/xlio_ultra_api_ping_pong.c

```diff
@@ -88,37 +117,57 @@ static struct app_state g_state = {0};
 #define USERDATA_SERVER_CLIENT 0xcafebabe
 #define USERDATA_CLIENT        0xfeedface
 
+/* TX buffer pool configuration */
+#define TX_BUFFER_COUNT 128 /* Number of TX buffers in the pool */
```

Comment:
if I'm a customer, I'll ask why 128? why 1024?
suggestion to make it clear:
`/* TX buffer pool configuration
 * These values are examples - adjust based on your application's:
 * - Message size requirements (TX_BUFFER_SIZE)
 * - Concurrent outstanding sends (TX_BUFFER_COUNT)
 */
#define TX_BUFFER_COUNT 128  /* Number of TX buffers in the pool */
#define TX_BUFFER_SIZE  1024 /* Size of each TX buffer in bytes */`

Reply:
done